src/edu/csus/ecs/pc2/ui/ContestInformationPane.java

```diff
@@ -213,7 +213,6 @@
     private JCheckBox shadowModeCheckbox;
 
     private JCheckBox allowMultipleTeamLoginsCheckbox;
-    private boolean allowMultipleLoginsPerTeam = false;
```

Comment:
Good change, allowMultipleLoginsPerTeam should not be used for MultipleTeamLogin state, the JCheckBox should be and is now used.